app/assets/javascripts/eventscripts/create_event.js

```diff
@@ -21,12 +21,65 @@ $( document ).on('turbolinks:load', function() {
         }
     });
 
-    $("#event_enddate").on("change", changeDur);
+    $("#event_enddate").on("change", adaptDuration);
 
+    $("#event_gameday_duration").on("change",durationChanged);
+
+    $("#event_game_mode").on("change",callChanged);
+
+    $("#event_max_teams").on("change",callChanged);
```

Comment:
This id is different depending on the event type. For Leagues its `league_game_mode` etc. so the method isnt being called.